File: Clase 8/ej4/JAVA/Main.java
public class Main{
    public static int contarCaracteres(String p){
        return p.length();
    }

    public static void main(String[] args){
        String palabra = "programacion"; 
        System.out.println("La palabra es: " + palabra); 

        System.out.println("La palabra tiene: " + contarCaracteres(palabra) + " letras");
    }
}

File: Clase 8/ej4/PYTHON/main.py
def contar_letras(p:str):
    #for i in range(p.__len__()):
    #    print(p[i])
    return p.__len__()

if __name__ == "__main__":
    palabra = "programacion"
    print(f"La palabra es: {palabra}")
    print(f"La palabra tiene {contar_letras(palabra)} letras")
